### PR TITLE
contrib: Update location of seeds.txt

### PR DESCRIPTION
**Pull request by laanwj**
Update the steps for updating the hardcoded seed nodes to point to the new filename on @sipa's server, and add command to decompress it.

Ref: #10163

### CODE REVIEW

**Comment by jonasschnelli** [comment]
Tested ACK ac9df130a507177ec77ef3f5f4fa43c0c5ef57c3

File: contrib/seeds/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ and remove old versions as necessary.
 
 The seeds compiled into the release are created from sipa's DNS seed data, like this:
 
-    curl -s http://bitcoin.sipa.be/seeds.txt > seeds_main.txt
+    curl -s http://bitcoin.sipa.be/seeds.txt.gz | gunzip > seeds_main.txt
```
Comment (practicalswift) [inline contrib/seeds/README.md]: Nit: Isn't `| gzip -dc` the preferred over `| gunzip`?

Comment (laanwj) [inline contrib/seeds/README.md]: For what reason?

Comment (practicalswift) [inline contrib/seeds/README.md]: @laanwj Good question! :-)  Perhaps it is a legacy thing, but for some reason I have it hard wired that `gzip -dc` is used when operating on stdin/stdout and `gunzip` is for unpacking `file.gz`, removing the original file and leaving `file`. They might be totally equivalent nowadays. If so, ignore my minor nit :-)

**Comment by paveljanik** [comment]
IIRC on some systems, gunzip symlink was not available in the default install.

File: contrib/seeds/README.md (hunk 1 above)
Comment (practicalswift) [inline contrib/seeds/README.md]: Found this thead: https://lists.gnu.org/archive/html/bug-automake/2009-10/msg00012.html

**Comment by laanwj** [comment]
> IIRC on some systems, gunzip symlink was not available in the default install.

I tried on FreeBSD, OpenBSD, Linux (Ubuntu) and `gunzip` worked out of the box at least... Also these are not general instructions for end-users, but for ourselves. I personally find `gunzip` much easier to remember. But meh I'm fine with changing it.

**Comment by fanquake** [comment]
utACK https://github.com/bitcoin/bitcoin/pull/10495/commits/ac9cd953d91e2448d24851b2f4973c7be8f4c3cd

**Comment by practicalswift** [comment]
ACK ac9cd95

**Pull request merged**